ui/packages/catalog-ui-search/src/main/webapp/component/search-form-interactions/search-form-interactions.view.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define, window*/
+
+
+
+define([
+    'wreqr',
+    'marionette',
+    'underscore',
+    'jquery',
+    './search-form-interactions.hbs',
+    'js/CustomElements',
+    'js/store',
+    'component/router/router',
+    'component/singletons/user-instance',
+    'component/loading/loading.view',
+    'component/lightbox/lightbox.view.instance',
+    'component/announcement',
+    'component/confirmation/confirmation.view'
+], function(wreqr, Marionette, _, $, template, CustomElements, store, router, user,
+    LoadingView, lightboxInstance, announcement, ConfirmationView) {
+    return Marionette.ItemView.extend({
+        template: template,
+        tagName: CustomElements.register('search-form-interactions'),
```

Comment:
Add `@import 'search-form-interactions/search-form-interactions';` to the `/ddf/ui/packages/catalog-ui-search/src/main/webapp/component/component.less` file and your CSS will get cleaned up.

Reply:
![screen shot 2018-04-03 at 9 43 17 am](https://user-images.githubusercontent.com/7055226/38263014-af414c9c-3723-11e8-89b0-271d7a9f5f4f.png)

Reply:
Registered the less file and display box now reflects the image above